modules/custom/external_comment/external_comment.module

```diff
@@ -25,3 +25,78 @@ function external_comment_query_comment_filter_alter(AlterableInterface $query)
     }
   }
 }
+
+
+
+function external_comment_cron() {
+  // fetch comments
+  $comment_ids = \Drupal::entityQuery('comment')
+    ->condition('entity_type', 'node')
+    ->condition('status', 1)
+    ->execute();
+
+  $comments_data = [];
+  // if comments exist
+  if ($comment_ids) {
+    $comments = \Drupal::entityTypeManager()
+      ->getStorage('comment')
+      ->loadMultiple($comment_ids);
+
+    foreach($comments as $comment) {
+      // Loop over and get fields for published comments
+      if ($comment->getStatus() == 1) {
+        $comments_data[] = [
+          'comment_id' => $comment->id(),
+          'uuid' => $comment->uuid(),
```

Comment:
this is comment entity's uuid, what you are looking for is content-type 'external'->field_uuid